imports/startup/server/index.js

```diff
@@ -16,4 +16,13 @@ Meteor.startup(() => {
     const newUrl = url.replace('#/', '');
     return `To reset your password, simply click the link below. ${newUrl}`;
   };
+  Accounts.emailTemplates.verifyEmail = {
+    subject() {
+      return 'Please verify your email address';
+    },
+    text(user, url) {
+      const newUrl = url.replace('#/', '');
+      return `Hi, To verify your account email, simply click the link below. ${newUrl}`;
+    },
+  };
 });
```

Comment:
I think we need to somehow handle the i18n in this, since we have swedish users who would want in swedish...